File: OCH_Tao/ABC/A/132.js
const S = [...(require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0]];
S.sort();
if (S[0] === S[1] && S[1] === S[2] && S[2] === S[3]) {
  console.log("No");
} else if (S[0] === S[1] && S[2] === S[3]) {
  console.log("Yes");
} else {
  console.log("No");
}

File: OCH_Tao/ABC/A/132.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  vector<char> S(4);
  for (int i = 0; i < 4; i++)
  {
    cin >> S[i];
  }
  sort(S.begin(), S.end());
  cout << (S[0] == S[1] && S[1] == S[2] && S[2] == S[3] ? "No" : S[0] == S[1] && S[2] == S[3] ? "Yes" : "No") << endl;
}